packages/hoppscotch-common/src/helpers/curl/sub_helpers/url.ts

```diff
@@ -53,7 +53,7 @@ const parseURL = (urlText: string | number) =>
       u
         .toString()
         .replace(/^'|'$/g, "")
-        .replaceAll(/[^a-zA-Z0-9_\-./?&=:@%+#,;()'\s]/g, "")
+        .replaceAll(/[^a-zA-Z0-9_\-./?&=:@%+#,;()'<>\s]/g, "")
```

Comment:
URL encoding seems coming into play. We could address this separately if this requires more effort.

https://github.com/user-attachments/assets/ee3515fa-007a-4480-b66f-b28ad5aada5e

Reply:
have pushed a [fix](https://github.com/hoppscotch/hoppscotch/pull/4180/commits/9fdb0420e077f9110d54de9359a3395ab706bff8) by decoding the URL using `decodeURIComponent`